auth.go

```diff
@@ -205,6 +206,7 @@ type Central struct {
 	// on their operands.
 	Stats                  CentralStats
 	authenticator          Authenticator
+	userstore              UserStore
```

Comment:
Use consistent casing for the variable name "userStore". Here it is "userstore", and elsewhere "userStore". I suggest sticking to "userStore" everywhere.